### DISCUSSION TITLE
Is Hue suitable for MySQL database?

### DISCUSSION THREAD

**Discussion by xulun2468** [cloudera/hue#3109] [Questions] [answered]
Hello team, may I ask if we are suitable for this product? I only have a mysql database. We only focus on making it easier for users to operate on the database, and can record additions, deletions, and changes to the database by different users.

**Comment by Harshg999** [comment, marked as answer]
Hey @xulun2468, Hue supports MySQL via `sqlalchemy` or `JDBC` interface. You should be able to perform all the operations for your use-case from the Hue editor for MySQL.

You can give it a try by following the below doc section to help you with the config setup: https://docs.gethue.com/administrator/configuration/connectors/#mysql

Hope it helps!

**Comment by xulun2468** [comment]
> Hey @xulun2468, Hue supports MySQL via `sqlalchemy` or `JDBC` interface. You should be able to perform all the operations for your use-case from the Hue editor for MySQL.
> 
> You can give it a try by following the below doc section to help you with the config setup: https://docs.gethue.com/administrator/configuration/connectors/#mysql
> 
> Hope it helps!

Thank you very much for your reply, I have another question, how do I know who deleted a certain database, where do I check these records

**Comment by Harshg999** [comment]
IIRC we have debug or info level logging of who executed statements via Hue and other audit details, that can help you identifying the user doing certain statement operations.

**Comment by liuzhu1995** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
